lib/modules/manager/npm/post-update/pnpm.spec.ts

```diff
@@ -77,4 +79,90 @@ describe('modules/manager/npm/post-update/pnpm', () => {
     expect(execSnapshots).toMatchSnapshot();
     // TODO: check docker preCommands
   });
+
+  it('uses constraint version if parent json has constraints', async () => {
+    const execSnapshots = mockExecAll(exec);
+    const configTemp = { cacheDir: 'some-cache-dir' };
+    const fileContent = Fixtures.get('parent/package.json');
+    fs.readFile = jest
+      .fn()
```

Comment:
@viceice looks like this approach is how the rest of the file is doing it, perhaps can we keep it as-is for now? Only npm and pnpm use this mock syntax, might be related to them using absolute fs directly instead of our util

Reply:
🤔 OK. Should be fixed soon in another PR